### ISSUE TITLE
Alteration: Remove Go-Karts' victory lap in race mode

### ISSUE THREAD

**Issue by Keatzee** [OpenRCT2/OpenRCT2#6206]
I think that the victory lap for winner in Go-Karts' race mode should be removed. There will certainly be opposition to this, so I'll explain my reasoning. It doesn't really serve a purpose, and slows down the lineup since peeps will have to wait until the victory lap is over before they can start the next race. This is evident on larger tracks, where the extra lap can waste an extra 30 seconds or more.

Of course I know there's continuous circuit mode, but that isn't as fun to watch, and it's limited to one lap per person.

As a compromise, I would support a checkbox in the operating settings to make the victory lap optional, but I think that would need the new save format. Or, as an "epic" suggestion, maybe the winner could get a special thought like "I won!" and a trophy or crown item. Just throwing stuff out there :stuck_out_tongue_winking_eye: 

Thoughts?

**Comment by Gymnasiast** [comment]
And how would the game know who won, if the winner can't pass the finish line?

**Comment by Keatzee** [comment]
That's a good point. What about putting the finish line at the end of the station?

**Comment by funzinnu** [comment]
good idea. However, it should be able to select an option.

**Comment by Gymnasiast** [comment]
Oh, please, no. We have way too many trivial options already.

**Comment by Keatzee** [comment]
And having an option would greatly complicate things for multiplayer. It would be either kept or removed.

**Comment by HaasJona** [comment]
It obviously wouldn't be an option in the option window. If this were to be implemented, I'd make it an additional operation mode of the ride, so you would have

* Race with victory lap
* Race without victory lap
* Continuous circuit

or something like that. I think that should be possible with the current save format, but obviously not high priority.

**Comment by Gymnasiast** [comment]
I still find it way too trivial, creating a separate ride mode just for something as minor as this.

**Comment by rwjuk** [comment]
I agree, and this is something that I imagine the scripting system will be able to handle - allowing those users who want this functionality to have it without modifying the core logic.

**Comment by IntelOrca** [comment]
Isn't this mostly only a problem with large circuits? The time it takes for the winner to go round is usually just enough for everyone else to switch seats with the next lot.

**Comment by Keatzee** [comment]
I'd call that a consensus... 😜 thanks to those who added to the discussion. Should this be closed with the veto tag?

**Comment by IntelOrca** [comment]
I think when we have a new save format, we could probably just add a checkbox to choose whether the winner goes round again. It would be simple to implement - we just can't extend the game like this until we have our own save format.
